frontend/src/scenes/session-recordings/SessionRecordings.tsx

```diff
@@ -215,24 +215,48 @@ function MainPanel(): JSX.Element {
     )
 }
 
+const ReplayPageTabs: ReplayTab[] = [
+    {
+        label: 'Recordings',
+        tooltipDocLink: 'https://posthog.com/docs/session-replay/tutorials',
+        key: ReplayTabs.Home,
+    },
+    {
+        label: 'Playlists',
+        tooltipDocLink: 'https://posthog.com/docs/session-replay/how-to-watch-recordings',
+        key: ReplayTabs.Playlists,
+        tooltip: 'View & create playlists',
+    },
+    {
+        label: 'Templates',
```

Comment:
This changed names, it used to be "Figure out what to watch". The old name sounded better tbh - it was clearer on what this is about, at least

Reply:
Good catch!